src/assertions.rs

```diff
@@ -126,6 +127,56 @@ macro_rules! assert_equal {
     }};
 }
 
+/// Asserts if left is greater than right.
+/// panics if they are not
+///
+/// ```rust
+/// use k9::assert_greater_than;
+///
+/// // simple values
+/// assert_greater_than!(2, 1);
+///
+/// #[derive(Debug, PartialOrd)]
+///
+/// let a1 = 15.75;
+/// let a2 = 10.32;
+///
+/// // this will print the visual difference between a1 & a2
+/// assert_greater_than!(a1, a2);
```

Comment:
from what i remember, the doc test will fail if a code block in here panics, is it not the case anymore?
https://doc.rust-lang.org/rustdoc/documentation-tests.html#passing-or-failing-a-doctest